internal/commands/list.go

```diff
@@ -48,23 +36,64 @@ func listCmd(dockerCli command.Cli) *cobra.Command {
 		},
 	}
 	cmd.Flags().StringVar(&opts.targetContext, "target-context", "", "List installations on this context")
+	cmd.Flags().BoolVar(&opts.allContexts, "all-contexts", false, "List installations on all contexts")
 
 	return cmd
 }
 
 func runList(dockerCli command.Cli, opts listOptions) error {
-	targetContext := getTargetContext(opts.targetContext, dockerCli.CurrentContext())
-	installations, err := getInstallations(targetContext, config.Dir())
-	if err != nil {
-		return err
+	var contexts []string
+	if opts.allContexts {
```

Comment:
Might be good to err if `--all-contexts` and `--target-context` are set

Reply:
Done 👍